library/x509_crt.c

```diff
@@ -1575,9 +1588,13 @@ int mbedtls_x509_crt_parse_path(mbedtls_x509_crt *chain, const char *path)
             continue;
         }
 
+        if (FAILED(SizeTToInt(wcslen(file_data.cFileName), &length_as_int))) {
+            return MBEDTLS_ERR_X509_FILE_IO_ERROR;
+        }
+
         w_ret = WideCharToMultiByte(CP_ACP, 0, file_data.cFileName,
-                                    -1,
-                                    p, (int) len,
+                                    length_as_int,
```

Comment:
Why can't we actually leave this as `-1`? That's for the case when the input string is null-terminated, which it should be, since we're using `wcslen()` to get the length for this parameter!

Reply:
Done as requested.